app/src/main/java/com/es/findsoccerplayers/MyPickers/DatePickerFragment.java

```diff
@@ -0,0 +1,40 @@
+package com.es.findsoccerplayers.MyPickers;
```

Comment:
Mi sembra che @giordy16 abbia creato una classe per fare la stessa cosa, quando ci si deve registrare e selezionare la data di nascita. Puoi usare quella classe

Reply:
No, in Register Activity ne ha creata una al volo. Io ho fatto le classi e poi usate. Infatti in register activity potremmo usare la mia DatePicker, ma lo faccio dopo.